src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1743,12 +1751,9 @@ public String toString() {
 			if (isNotBlank(getMatrixCoefficients())) {
 				result.append(", Matrix Coefficients: ").append(getMatrixCoefficients());
 			}
-			if (isNotBlank(avcLevel)) {
-				result.append(", AVC Level: ").append(getAvcLevel());
+			if (isNotBlank(getVideoFormatProfile())) {
+				result.append(", Video Format Profile: ").append(getVideoFormatProfile());
```

Comment:
@Sami32 Maybe the name is confusing, but I used the term from MediaInfo which is ```format_profile```. It's the whole string, not just the profile or just the level. It's the complete string as returned from MediaInfo, e.g ```High@l4.1```.